backend/de.metas.banking/de.metas.banking.base/src/main/java/de/metas/banking/callout/C_BankStatementLine.java

```diff
@@ -136,11 +115,24 @@ public void onLink_BankStatement_IDChangedResetAmounts(final I_C_BankStatementLi
 	}
 
 	@CalloutMethod(columnNames = I_C_BankStatementLine.COLUMNNAME_C_BP_BankAccountTo_ID)
-	public void onC_BP_BankAccountTo_IDChanged(final I_C_BankStatementLine bsl)
+	public void onC_BP_BankAccountTo_IDChanged(final @NonNull I_C_BankStatementLine bsl)
 	{
 		if (bsl.getC_BP_BankAccountTo_ID() <= 0)
 		{
 			bsl.setLink_BankStatementLine_ID(0);
 		}
 	}
+
+	@CalloutMethod(columnNames = I_C_BankStatementLine.COLUMNNAME_C_Invoice_ID)
```

Comment:
it's OK to follow the naming convention within this class, but otherwise please prefer a method name that descripbes what the method does, rather than when it's invoked.